cfn-guard/tests/functional.rs

```diff
@@ -814,6 +814,45 @@ AWS::EC2::Volume Size == 101 |OR| AWS::EC2::Volume Size == 99"#,
         );
     }
 
+    #[test]
+    fn test_or_should_pass_numeric() {
```

Comment:
I believe this is confirming that it fails correctly.  If so, can you update the test name to indicate that?

Please update the test with multiple smaller internal `assert_eq!` to check that it fails correctly for each individual operator (the `|OR|` here could obscure a bad path somewhere).

Also, please add tests for:
1.  Showing that each of the new operators passes correctly
2.  That it handles a non-numeric value being used in a numeric comparison as a runtime error that exits with a `1` and returns a message with the text from the rule that failed.